app/src/test/java/com/fcaputo/parkingapp/DateTimeCustomFormatTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.fcaputo.parkingapp
+
+import com.fcaputo.parkingapp.utils.DateTimeCustomFormat
+import org.junit.Assert
+import org.junit.Test
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.Month
+import java.time.ZoneOffset
+import java.util.Calendar
+import java.util.Date
+
+class DateTimeCustomFormatTest {
+    @Test
+    fun `when a valid date string is passed, the formatter parses it correctly`() {
+        val parsedDate = DateTimeCustomFormat.parseDate(VALID_DATE_STRING)
+        Assert.assertEquals(VALID_DATE.year, parsedDate[Calendar.YEAR])
+        Assert.assertEquals(VALID_DATE.monthValue - 1, parsedDate[Calendar.MONTH])
```

Comment:
do not hardcode

Reply:
Yes I forgot to refactor this test, since I was unifying all date types to Calendar
Once I am done doing this, that hardcode should disappear

Reply:
Done